config/auto-doc.php

```diff
@@ -100,7 +100,7 @@
     |
     | The name of driver, which will collect and save documentation
     | Feel free to use your own driver class which should be inherited from
-    | `RonasIT\Support\AutoDoc\Interfaces\SwaggerDriverInterface` interface,
```

Comment:
I believe we shouldn't force developers to use BaseDriver as the parent class.

Let's just make BaseDriver implementing SwaggerDriverInterface interface